src/components/UsersForm/UsersForm.jsx

```diff
@@ -66,6 +75,11 @@ class UsersForm extends PureComponent {
           </DialogContent>
 
           <DialogActions>
+            <DeleteMember
+              user={user}
+              refetch={refetch}
+              closeDialog={onRequestClose}
```

Comment:
Is this `closeDialog` prop necessary? I don't see `onRequestClose` defined in this component.

Reply:
Its used in the `DeleteMember` compnent to close the dialog after the confirming.